Content.Client/Administration/UI/CustomControls/ConfirmWindow.xaml.cs

```diff
@@ -0,0 +1,36 @@
+using Content.Client.Stylesheets;
+using System.Numerics;
+using Robust.Client.AutoGenerated;
+using Robust.Client.Console;
+using Robust.Client.UserInterface.CustomControls;
+using Robust.Client.UserInterface.XAML;
+
+namespace Content.Client.Administration.UI.CustomControls;
+
+[GenerateTypedNameReferences]
+public sealed partial class ConfirmWindow : DefaultWindow
```

Comment:
The layout of this window isn't great. Image search for examples of confirmation dialogues to get an idea of what's more typical.

I'd suggest having a constant title of simply "Confirm" and, instead of just listing the command name, add an additional label to the window with a specific question, e.g. "This will restart the round. Are you sure you wish to do this?" - you can repurpose the `RequiresConfirm` to store that string - just rename it to ConfirmationPrompt or something similar.